internal/linter/config/files.go

```diff
@@ -0,0 +1,17 @@
+package config
+
+// Directories represents the target directories.
```

Comment:
@wangfenjin You need to fix this comment, and it will pass CI.

```
/go/src/github.com/yoheimuta/protolint/internal/linter/config/files.go:3:1: comment on exported type Files should be of the form "Files ..." (with optional leading article)
Found 1 lint suggestions; failing.
Makefile:17: recipe for target 'test/lint' failed
```